integration-tests/src/test/resources/features/slackaction.feature

```diff
@@ -0,0 +1,93 @@
+Feature: Slack Action tests
+
+  Background:
+    Given authenticate against Manager
+    And create a new Bridge "mybridge"
+    And the Bridge "mybridge" is existing with status "ready" within 2 minutes
+    And the Ingress of Bridge "mybridge" is available within 2 minutes
+
+  Scenario: Slack Action Processor is created and slack message should be received in the slack channel
+    When add a Processor to the Bridge "mybridge" with body:
+    """
+    {
+      "name": "slackProcessor",
+      "action": {
+        "type": "Slack",
+        "parameters": {
+            "channel": "mc",
```

Comment:
May be good to set channel in pom file, as it may change on different environment.